src/components/UpdateNotificationDialog.vue

```diff
@@ -70,6 +80,14 @@ const emit =
     (e: "update:modelValue", value: boolean): void;
   }>();
 
+const skipThisVersion = ref<boolean>(false);
+
+const setSkipVersion = () => {
+  if (skipThisVersion.value) {
+    window.electron.setSetting("skipUpdateVersion", props.latestVersion);
```

Comment:
（実は）window.electronはダイレクトに呼ばないようにしていきたい方針なので、ちょっとコメントだけ！
```suggestion
    // FIXME: window.electronを直に呼ばないようにする
    window.electron.setSetting("skipUpdateVersion", props.latestVersion);
```
（実はダイレクトに呼んでもそんなに問題ないんじゃないかとちょっと最近思ってますが）